starter-workflows/code-scanning/scorecards.yml

```diff
@@ -1,10 +1,25 @@
+# Copyright 2022 OpenSSF Authors
```

Comment:
you can remove this file if you want. It out of date and we're not maintaining it. It was supposed to be a local copy of the workflow we give to GitHub's starter-workflow, but we just get it from https://github.com/actions/starter-workflows/blob/main/code-scanning/scorecards.yml directly instead.

Reply:
Opened a separate issue to discuss: https://github.com/ossf/scorecard-action/issues/315